src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
@azaroth42 @jpstroop Thank you, next week is fine! 

The code prior to this patch accounted for the fact there can be different (square) sizes at different levels; the patch hopefully maintains that properly, but I wanted to have your eyes on it to make sure. 

It looks like that info.json has square tiles...is it possible with IIIF to have non-square tiles? If so should we add support for that in this patch?